src/test/java/com/revature/hydra/curriculum/service/tests/CurriculumTests.java

```diff
@@ -0,0 +1,18 @@
+package com.revature.hydra.curriculum.service.tests;
+
+public class CurriculumTests {
+
+//	@Test
+//	public void testGetAllCurriculums() {
```

Comment:
Why are tests being commented out❓

Reply:
they're old stuff; they'll be removed in a later push